lang/extract_json_strings.py

```diff
@@ -205,6 +205,7 @@ def extract_harvest(item):
 
 def extract_bodypart(item):
     outfile = get_outfile("bodypart")
+    writestr(outfile, item["name"])
```

Comment:
Unless I'm missing something this is unrelated to the rest of the PR.
it's also redundant with the next line.